### PR TITLE
Add Polish language

### PR DESCRIPTION
**Pull request by jakubdrozdek**
I'd like to contribute by adding Polish language to the list. Wish to co-operate with any other volunteers.

I've got experience in English->Polish translation - I used to be a non-profit translator in [IronSquad team](http://www.ironsquad.pl/). "Worked" there for 5 years, mostly translating computer games (Borderlands 2, Aion, Brutal Legend, etc.). Also, I used to be an editor, before the team shutdown.

I've been using React since 15.x and I think I know quite a bit about it :-)

I don't have experience in open source projects, though. This is my first contribution ever; hopefully not the last one :-)

### CODE REVIEW

**Comment by bartlomiejzuber** [comment]
Hello, I will be glad to help you with translation.

I've been translator of polish wow wikia during my school days.
Here's link to my contribution there: https://wowwiki.fandom.com/pl/wiki/Specjalna:Wk%C5%82ad/Bajtas

Same as @jakubdrozdek I've been using React since 15.x making with it hundreds of sites and web apps. In mean time I am giving speeches about mastering React/JS in local companies.

One of my 2019 main goal is to become active open source contributor. That's my chance. 😄

File: langs/pl.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Polish",
+    "code": "pl",
+    "maintainers": [
+        "jakubdrozdek"
```
Comment (bartlomiejzuber) [inline langs/pl.json]: @jakubdrozdek Please add me to the list. 🙏

**Comment by TeoTN** [comment]
Hi,
I can help with the translation. Despite little experience with translating (I was contributing to some open-source crowd translation platform, I don't have a link though), I have a Certificate in Advanced English, and I'm also tech speaker & presenter leading various tech talks / workshops, so I'm familiar with technical terms and their translations, and I can explain technical nuances in a grammatically correct way. Of course, this is mostly about keeping fidelity of the translation.
Cheers!

**Comment by tesseralis** [comment]
@jakubdrozdek if you're okay with it, I'll add the other two and start you off!

**Comment by jakubdrozdek** [comment]
@tesseralis Yeah, go ahead, if want to do it yourself :-)
When and how can we start?

**Comment by tesseralis** [comment]
Okay merged! You should get an email to join the reactjs organization. I'll link you the repo once it's created :)

**Comment by tesseralis** [comment]
Here's your repo! https://github.com/reactjs/pl.reactjs.org/issues/1

**Comment by cvqprs** [comment]
Guys, I can help with a translation. Can u add me to the contributors pls and I can start today ;-)

**Pull request merged**